### PR TITLE
[Tree widget & Property grid]: Run prettier to format source code

### PR DESCRIPTION
**Pull request by saskliutas**
Ran prettier in property-grid and tree-widget packages to unify code formatting.

### CODE REVIEW

File: packages/itwin/property-grid/.eslintrc.js [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
     "object-curly-spacing": ["error", "always"],
-    "@typescript-eslint/no-unused-vars": "off",
-    "unused-imports/no-unused-imports": "error",
-    "unused-imports/no-unused-vars": [
-      "error",
-      { "vars": "all", "varsIgnorePattern": "^_", "args": "after-used", "argsIgnorePattern": "^_" }
-    ]
-  }
-}
+
```
Comment (grigasp) [inline packages/itwin/property-grid/.eslintrc.js:15]: why remove the `no-unused-*` rules?

File: packages/itwin/property-grid/tsconfig.json [hunk 2]
```diff
@@ -8,5 +8,5 @@
     "resolveJsonModule": true,
     "lib": ["DOM", "ES2019"]
   },
-  "include": ["src"],
+  "include": ["src", "playwright.config.ts"]
```
Comment (grigasp) [inline packages/itwin/property-grid/tsconfig.json:11]: this will build and include build output of `playwright.config.ts` into the `lib`, won't it?

Comment (saskliutas) [reply packages/itwin/property-grid/tsconfig.json]: Removed

File: packages/itwin/property-grid/.eslintrc.js (hunk 1 above)
Comment (saskliutas) [reply packages/itwin/property-grid/.eslintrc.js]: Forgot to revert after testing. Added back.

**Review by grigasp — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]

**Pull request merged**